### ISSUE TITLE
Maze Map Walls Invisibile Levels 1-3

### ISSUE THREAD

**Issue by jerkyspace** [jessaskey/mhavocpe#114]
0.52 romset
0.212 HBMAME

The maze map at the top of the screen isn't visible on levels 1-3 but comes back on level 4. You can see the enemies and objects but no walls.It also blanks out the Oxygen and Reactor time values.

When I go back to other levels using save states (the next day) the same thing happens on different levels (other than 1-3).

After you play another level or few the borders and numbers come back on a subsequent maze. Sometimes it's one level, sometimes it takes 3 or more.

**Comment by jessaskey** [comment]
I can't seem to see this issue in 0.53 (my current build) and since I cheated and had a couple different versions of 0.52 out there to you, it may just be an issue with that particular 0.52 build that I put out there last. I can confirm that in 0.53 the map is there and the Oxygen and Reactor times are good, so Im going to just assign this to 0.53 and guess that it will be fixed in this next build.

**Comment by jerkyspace** [comment]
I just noticed that on levels where the maze scanner borders and values for o2 and reactor time are missing...the o2 values are wrong or counting down too fast somehow (which I cannot actually see but am definitely running out way earlier than I should).

I first noticed this on level 20 in my recent play testing session that when I set off the reactor I didn't have enough o2 to go back around and get the "bonus" oxygen objects in the middle of the maze I allowed access too via that dynamic wall by the reactor. 

Then on Level 21 and 22 the Oxygen values were also blanked out and I kept running out of o2 near the end of level 22 on the way to the Cyan key. I could not get back to the Reactor from there without choking out.

I upgraded to HBMAME 0.212 just before running into all these issues so I am going to go back to 0.209 and run through this romset from Level 1 again to double check things.
